pkg/lib/config/testdata/parse_secret_tests.yaml

```diff
@@ -188,4 +188,24 @@ config:
   secrets:
     - key: bot_protection.provider
       data:
-        type: hcaptcha # not supported
\ No newline at end of file
+        type: hcaptcha # not supported
+---
+name: ldap/valid
+error: null
+config:
+  secrets:
+    - key: ldap
+      data:
+        name: test
+        username: user
+        password: secret
+---
+name: ldap/missing-required-secret-key
+error: |-
+  invalid secrets:
+  /secrets/0/data: required
+    map[actual:<nil> expected:[name] missing:[name]]
+config:
+  secrets:
+    - key: ldap
+      data: {}
```

Comment:
Try to be specific in the commit message.

> Update secret test yaml 

What are you updating? Everyone knows you are updating secret test yaml but for what.